Add a rake task to populate a development database.

jfhamlin opened the pull request:
This includes the commit from #16. We'll want to wrap that PR up and get it in before merging this one.

jfhamlin commented on lib/tasks/populate.rake:
```diff
@@ -0,0 +1,35 @@
+
+    128.times do
+      name = Faker::Company.name
+      short_description = Faker::Lorem.sentence if rand(2) == 0
```
We could also just add logic like this to the base factory.

lelandg559 commented:
Ocassionally seeing this because of random duplicate on unique index name

leland@ophion:~/askdarcel-api$ rake db:populate
rake aborted!
ActiveRecord::RecordNotUnique: PG::UniqueViolation: ERROR:  duplicate key value violates unique constraint "index_categories_on_name"
DETAIL:  Key (name)=(.) already exists.
: INSERT INTO "categories" ("name", "created_at", "updated_at") VALUES ($1, $2, $3) RETURNING "id"
/Users/leland/askdarcel-api/lib/tasks/populate.rake:12:in `block (3 levels) in <top (required)>'
/Users/leland/askdarcel-api/lib/tasks/populate.rake:11:in `times'
/Users/leland/askdarcel-api/lib/tasks/populate.rake:11:in `block (2 levels) in <top (required)>'
PG::UniqueViolation: ERROR:  duplicate key value violates unique constraint "index_categories_on_name"
DETAIL:  Key (name)=(.) already exists.
/Users/leland/askdarcel-api/lib/tasks/populate.rake:12:in `block (3 levels) in <top (required)>'
/Users/leland/askdarcel-api/lib/tasks/populate.rake:11:in `times'
/Users/leland/askdarcel-api/lib/tasks/populate.rake:11:in `block (2 levels) in <top (required)>'
Tasks: TOP => db:populate
(See full trace by running task with --trace)




Other issue is that the join table isn't being appropriately filled up.

jfhamlin commented:
> Ocassionally seeing this because of random duplicate on unique index name

Yeah, I originally just let that be, but it'll get annoying, so I've fixed that.

> Other issue is that the join table isn't being appropriately filled up.

Good catch. Fixed that too.

lelandg559 commented:
yeap LGTM
